setup.py

```diff
@@ -0,0 +1,15 @@
+from setuptools import find_packages, setup
+
+version = '0.0.3'
```

Comment:
```suggestion
version = '0.0.5'
```

Reply:
@lee1043 - my plan is to merge this to main and increment the version number again.